code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -377,6 +383,21 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 /obj/item/stack/sheet/mineral/wood/fifty
 	amount = 50
 
+/obj/item/stack/sheet/mineral/wood/interact_with_atom(mob/living/carbon/human/H, mob/user)
```

Comment:
Don't use single letter variables, give it a real name.

Reply:
fixed